Runtime/Data/Attribute/RestrictedAttribute.cs

```diff
@@ -37,7 +45,7 @@ public enum Restrictions
         /// Draws the property in a specified restricted way.
         /// </summary>
         /// <param name="restrictions">The restriction options to apply.</param>
-        public RestrictedAttribute(Restrictions restrictions = Restrictions.ReadOnlyWhenPlayingAndEnabled | Restrictions.Muted)
+        public RestrictedAttribute(Restrictions restrictions = Restrictions.ReadOnlyAtRunTime | Restrictions.Muted)
```

Comment:
Shouldn't the default be `ReadOnlyAtRunTimeAndEnabled`?

Reply:
Well that may not be the default required state. The default for our obs lists for example would be only readonly when it's running and disabled (because you can't make changes when disabled)

but then there are times where you may only want to make changes when enabled.

So in the end i thought it's probably best to just default handle both being considered the default, besides the existing use case of [Restricted] which is [InternalSetting] is for our facade settings, which shouldn't be changed at all at runtime. So i settled on ReadOnlyAtRunTime being the default (as it's going to be used pretty much everywhere in VRTK)

Reply:
👍